CheckManga.egg-info/top_level.txt

```diff
@@ -0,0 +1 @@
+CheckManga
```

Comment:
can egg-info be in gitignore or nah? you could always have a readme explaining how to install egg info via terminal if its easy. o/w best keep this stuff in its own PR